R/utilities.R

```diff
@@ -2910,3 +2910,310 @@ adjust_ploidy = function(this_seg,
 
   return(this_seg)
 }
+
+
+#' This function will compare CNV data between samples with multiple time points. It can also handle same-sample comparison
```

Comment:
This format will put this entire block of text into the title so when you look at the help for this function all these details are in large bold font at the top. The format should go like this: 

```
#' Sum of vector elements
#'
#' `sum` returns the sum of all the values present in its arguments.
#'
#' This is a generic function: methods can be defined for it directly
#' or via the [Summary()] group generic. For this to work properly,
#' the arguments `...` should be unnamed, and dispatch is on the
#' first argument.
```

(From https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html)

Perhaps while you fix this you could update the docstrings for `sanitize_maf_data`?

Reply:
Thanks, I'll fix this!